src/api/api.js

```diff
@@ -4,13 +4,25 @@ import {
   transformTimeSeries,
   transformHourly,
   transformSearchResults,
+  transformLocationInfo,
 } from './transforms';
 
 // -------------
 // API Calls
 // -------------
 
 /**
+ * Get information for a location
+ *
+ * @param {String} locationId The location to query (e.g., nauswaseattle)
+ * @return {Promise} A promise after the get request was made
+ */
+export function getLocationInfo(locationId) {
+  return get(`/locations/${locationId}`)
```

Comment:
Oh I didn't see that one. API docs seems to suggest it is identical to this, but you guys have liked ending with some descriptor so I can switch.